### PR TITLE
[InProgress]up-and-going-#ch2

### PR DESCRIPTION
**Pull request by memoac**

### CODE REVIEW

**Comment by EstebanMarin** [comment]
Hey thanks for your time!!!! as soon as you  are ready please let us know so we can add you as a contributor in the README.md file!!

File: up & going/ch2.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ Introduciremos una serie de conceptos en este capítulo que no seran totalmente
 
 We will introduce quite a few concepts in this chapter that will not be fully explored until subsequent *YDKJS* books. You can think of this chapter as an overview of the topics covered in detail throughout the rest of this series.
 
+Especialmente si usted es nuevo en Javascript, debería esperar gastar un poco más de tiempo repasando los conceptos y ejemplos de código en este libro varias veces. Como toda buena base, ésta se va construyendo poco a poco, no espere a que todo lo aprenda inmediatamente al primer intento.
+
```
Comment (giovanymoreno) [inline up & going/ch2.md]: Traducido en https://github.com/You-Dont-Know-JS-ES/Traduccion/pull/47

**Comment by EstebanMarin** [comment]
@giovanymoreno makes sense. @memoac you can continue your translation with other paragraphs

**Comment by mballeng91** [comment]
@memoac Hey bro! Are you going to continue working on this book section?

**Pull request closed without merging**